src/participant/participant.ts

```diff
@@ -1201,6 +1200,81 @@ export default class ParticipantController {
     });
   }
 
+  async exportCodeToPlayground(): Promise<boolean> {
+    const activeTextEditor = vscode.window.activeTextEditor;
+    if (!activeTextEditor) {
+      await vscode.window.showErrorMessage('Active editor not found.');
+      return Promise.resolve(false);
```

Comment:
We can return the value here and avoid the promise wrapper. (Two more instances below)
```suggestion
      return false;
```